src/components/saas-list/saas-list.component.ts

```diff
@@ -0,0 +1,10 @@
+import {SaasListController} from './saas-list.controller';
+
+export const saasList: angular.IComponentOptions = {
+  bindings: {
+    title: '<?',
```

Comment:
I think I like it better when title was outside the component, particularly if the page embedding might change it. Any compelling reason to move it here?